frontend/components/Archives.tsx

```diff
@@ -123,6 +124,7 @@ const Archives = ({ images }: { images: string[] }) => {
           icon={<FaArrowCircleLeft />}
           onClick={prevSlide}
           aria-label={"Previous"}
+          isDisabled={startIndex <= 0}
```

Comment:
Should we remove the arrows instead of disabeling them?

Reply:
Not sure as that might shift view around

Reply:
what if you have a fixed width for the arrow container (minW) and just remove or show the arrow depending of the situation.
It may be annoying to click on the arrow when there isn't more archives

Reply:
I think the colour and hover state should explain that instead of randomly removing elements from the screen but happy to hide it anyway

Reply:
I think it may be bad UI having clickable items that are not clickeable

Reply:
we can just disable the arrows and have greyed out state when no scroll available. i believe that's typically what is done

Reply:
what @democrazyfi currently has is good enough imo. can do a final clean up in very end before we go live if we need to correct any UI things. we could debate them on a call together

Reply:
yeah most chakra components handle a disabled (greyed out) state as you can see here

<img width="496" alt="Screenshot 2024-04-02 at 19 47 21" src="https://github.com/yearone-io/burntpix-archives/assets/132473036/fda8963e-34c3-4239-909b-fb8e5a4bf286">